fetch.bs

```diff
@@ -3089,6 +3122,11 @@ steps:
   a body, have a bounded size and are not allowed to stay alive indefinitely.
 
 
+ <li><p>If <var>httpRequest</var>'s <a for=request>body</a> is non-null and <var>httpRequest</var>'s
+ <a for=request>body</a>'s <a for=request>source</a> is null, then <a for="header list">append</a>
+ `<code>Transfer-Encoding</code>`/`<code>chunked</code>` to <var>httpRequest</var>'s
+ <a for=request>header list</a>.
```

Comment:
Fixed though I'm not sure "_connection_ is not an HTTP/2 connection" is the right wording.